liveRecorder/Tasks/ConcatinationTask.py

```diff
@@ -5,6 +5,7 @@
 import re
```

Comment:
why this task only? why not ALL a logic that belong on the BaseTask?

Reply:
The first task that get new job is ConcatenationTask. The retries, can be global for job handling cycle or per task in job handling cycle. We didn't discuss it so I chose to relate retires to whole job cycle. Let's discuss it.

Reply:
yes.. but you can have the same problem in the upload task?

Reply:
Anyway I moved logic to TaskBase. In the first place the retries counter was used for ALL jobs created on the entry. 
I suggest to decide which errors should have retries and which not, not just whether every task incoming should reset the retries. This will reduce useless reprocessing, in case of unrecoverable errors.